packages/opentelemetry/src/spanExporter.ts

```diff
@@ -215,33 +225,46 @@ function createAndFinishSpanForOtelSpan(node: SpanNode, sentryParentSpan: Sentry
   // If this span should be dropped, we still want to create spans for the children of this
   if (shouldDrop) {
     node.children.forEach(child => {
-      createAndFinishSpanForOtelSpan(child, sentryParentSpan, remaining);
+      createAndFinishSpanForOtelSpan(child, spans, remaining);
     });
     return;
   }
 
-  const spanId = span.spanContext().spanId;
-  const { attributes } = span;
+  const span_id = span.spanContext().spanId;
+  const trace_id = span.spanContext().traceId;
 
-  const { op, description, data, origin } = getSpanData(span);
-  const allData = { ...removeSentryAttributes(attributes), ...data };
+  const { attributes, startTime, endTime, parentSpanId } = span;
 
-  // eslint-disable-next-line deprecation/deprecation
-  const sentrySpan = sentryParentSpan.startChild({
-    name: description,
-    op,
+  const { op, description, data, origin = 'manual' } = getSpanData(span);
+  const allData = dropUndefinedKeys({
+    [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: origin,
+    [SEMANTIC_ATTRIBUTE_SENTRY_OP]: op,
+    ...removeSentryAttributes(attributes),
+    ...data,
+  });
+
+  const status = mapStatus(span);
+
+  const spanJSON: SpanJSON = dropUndefinedKeys({
```

Comment:
Just wanna make sure: Are all of these calls to `dropUndefinedKeys` necessary?

Reply:
🤷 I added them because tests failed because these were now on the event, so I think it's the safe bet to remove them?

Reply:
hm. If it is reasonable to do I would check if the tests are testing the right thing. Otherwise this is fine for now. The only concern is that if we don't do it now, we will do it never because we'll forget. Right now its not an issue but as soon as we use otel in the browser this will be a bundle size concern.